File: backend/src/main/java/ipeps/pwd/wallet/entity/payload/EmployeeCreatePayload.java
package ipeps.pwd.wallet.entity.payload;


import ipeps.pwd.wallet.entity.Company;
import ipeps.pwd.wallet.entity.enums.EmployeeStatus;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

import javax.validation.constraints.NotNull;


@Data
@NoArgsConstructor
@AllArgsConstructor
public class EmployeeCreatePayload {
     String Lastname;
     String Firstname;
     String Address;
     String Gender;
     String Birthday;
     String Ssin;
     EmployeeStatus status;
     Company company;
     Boolean active;
     String City;
     String Phone;
     String Email;
     String Picture;

}

File: WalletApp/src/app/modules/dashboard/component/employee/payload/EmployeeCreatePayload.ts
import {EmployeeStatus} from "../model/enum/EmployeeStatus";
import {Company} from "../../company/model/business/company";

export interface EmployeeCreatePayload{
  Lastname : string;
  Firstname: string;
  Address: string;
  Gender: string;
  Birthday: string;
  Ssin: string;
  status : EmployeeStatus;
  company: Company;
  active : boolean
}
